pages/featured.jsx

```diff
@@ -1,10 +1,7 @@
 import React from 'react';
-import ProjectList from '../components/project-list/project-list.jsx';
+import ProjectLoader from '../components/project-loader/project-loader.jsx';
+
+export default function () {
+  return <div><ProjectLoader params={{featured: `True`}} /></div>;
```

Comment:
why are we not bootstrapping as `<ProjectLoader featured={true} />`? Passing a string into an object that is bound to "params" feels way too many levels of indirection.